src/frontend-pwa/src/views/Report/report.styles.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @summary This is the styling page for the Report view screen.
+ * @author  Tyler Maloney
+ * */
+
+import styled from '@emotion/styled';
+import typography from '../../typography';
+
+export const StyledReportOuterDiv = styled.div`
+    width: 100%;
+    min-height: 100svh;
+    // position: absolute;
```

Comment:
Try not to push commented out code. This often gets missed later and leads to clutter.